### PR TITLE
Two more tests for the staggered scheme for HT process

### PR DESCRIPTION
**Pull request by wenqing**
as titled.

While an old test is removed.

### CODE REVIEW

File: web/content/docs/benchmarks/hydro-thermal/decovalex-TH.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+ in ogs6. The simplifications are
+<ol>
+<li><p>resizing the domain to an area that can represent the near field of an
+ installed nuclear water canister,</p></li>
+<li><p>assuming the the betonite is fully saturated from the beginning,</p></li>
+<li><p>ignoring the mechnical process.</p></li>
+</ol>
+<p>With such simplifications, the geometry of the present example is illustrated
+ in the following figure:</p>
+<p><img src="decovalex_TH_domain.png" alt="image" /></p>
```
Comment (endJunction) [inline web/content/docs/benchmarks/hydro-thermal/decovalex-TH.md]: Looking at the build web pages it seems the images are missing. https://jenkins.opengeosys.org/job/ufz/job/ogs/job/PR-2076/4/Web/

File: web/content/docs/benchmarks/hydro-thermal/decovalex-TH.md [hunk 2]
```diff
@@ -0,0 +1,131 @@
++++
+project = "Parabolic/HT/StaggeredCoupling/ADecovalexTHMCBasedHTExample/th_decovalex.prj"
+author = "Wenqing Wang"
+date = "2018-02-13T12:42:53+01:00"
+title = "An example about the coupled thermal hydraulic processes in the FEBEX type repository"
```
[the remark this answers is not in the document]
Comment (wenqing) [reply web/content/docs/benchmarks/hydro-thermal/decovalex-TH.md]: Changed `about` to `of` .

File: web/content/docs/benchmarks/hydro-thermal/decovalex-TH.md (hunk 1 above)
Comment (wenqing) [reply web/content/docs/benchmarks/hydro-thermal/decovalex-TH.md]: Tried by using the following image including syntax:
```
{{< img src="../decovalex_TH_domain_pT.png" > }}
```

Comment (wenqing) [reply web/content/docs/benchmarks/hydro-thermal/decovalex-TH.md]: It works.

**Pull request merged**